src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -213,15 +213,31 @@ bool clang_cpp_convertert::get_var(const clang::VarDecl &vd, exprt &new_expr)
   return clang_c_convertert::get_var(vd, new_expr);
 }
 
-bool clang_cpp_convertert::get_function(
+bool clang_cpp_convertert::get_method(
   const clang::FunctionDecl &fd,
   exprt &new_expr)
 {
   // Only convert instantiated functions/methods not depending on a template parameter
   if(fd.isDependentContext())
     return false;
 
-  if(clang_c_convertert::get_function(fd, new_expr))
+  // Don't convert if implicit, unless it's a constructor/destructor/
+  // Copy assignment Operator/Move assignment Operator
+  // A compiler-generated default ctor/dtor is considered implicit, but we have
+  // to parse it.
+  if(
+    fd.isImplicit() &&
+    !is_ConstructorOrDestructor(fd) /*&& !is_CopyOrMoveOperator(fd)*/)
```

Comment:
maybe you forgot to uncomment the if condition?

Reply:
In fact, the compiler generated the operator with the unnamed argument, which resulted in invalid Pointers (as I mentioned in the issue), and we needed to tweak the operator a bit.